lib/routes/gov/zj/ningbogzw-notice.ts

```diff
@@ -0,0 +1,50 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/zj/ningbogzw-notice/:colId?',
+    categories: ['government'],
+    example: '/gov/zj/ningbogzw-notice/1229116730',
+    parameters: {
+        colId: '公告分类id、详细信息点击源网站http://gzw.ningbo.gov.cn/请求中寻找',
+    },
+    radar: [
+        {
+            source: ['gzw.ningbo.gov.cn/col/col1229116730/index.html'],
+            target: '/zj/ningbogzw-notice/:colId?',
+        },
+    ],
+    name: '宁波市国资委-公告',
+    url: 'gzw.ningbo.gov.cn',
+    maintainers: ['HaoyuLee'],
+    description: `
+    | 公告类别         | colId |
+    | ------------ | -- |
+    | 首页-市属国企招聘信息-招聘公告     | 1229116730  |
```

Comment:
The description is still indenting with 4 spaces.
https://github.com/DIYgod/RSSHub/pull/16404#discussion_r1711671066